index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <div>
+        <header class="header flex-row">
+            <div class="menu-button">
+                <a class="logo">JEPM</a>
+            </div>
+            <nav class="menu-mobile">
+                <button type="button" class="menu-button"><img src="./img/header-menu.png" alt="menu-image"></button>
+            </nav>
+        </header>
+        <section class="headline main">
+            <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                Glad to see you!<br>
+            </h1>
+            <p class="color-text">I'm a software developer! I can help you build a product, feature or website Look
+                through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don't hesitate to contact me.</p>
+            <div class="text-connect">
+                <a>LET'S CONNECT</a>
+            </div>
+            <ul class="icons-socialMedia flex-row">
+                <li class="icon1"><a href="https://twitter.com/praduar"><img src="./img/twitter1.png"
+                            alt="twitter-icon"></a></li>
+                <li class="icon2"><a href="https://www.linkedin.com/in/jorgepradamoncada/"><img src="./img/linkedin.png"
+                            alt="linkedin-icon"></a></li>
+                <li class="icon3"><a href="https://medium.com/" rel="noopener noreferrer"><img src="./img/medium.png"
+                            alt="medium-icon"></a></li>
+                <li class="icon2"><a href="https://github.com/pradmon"><img src="./img/github.png"
+                            alt="github-icon"></a>
+                </li>
+                <li class="icon2"><a href="https://www.angellist.com/"><img src="./img/angellist.png"
+                            alt="angellist-icon"></a></li>
+            </ul>
+        </section>
```

Comment:
- [ ] As required for the project, kindly make sure your `Headline` section sticks as much as possible to the one of your chosen design. e.g.: `text-color`, `fonts(size, weight,...)`, `spacing`(around & between), `margin`, `padding`,...

| Figm design | Your Design |
| ------------------ | ------------- |
| ![Screenshot from 2023-04-27 06-23-53](https://user-images.githubusercontent.com/83042742/234759825-cc0d2b35-8ed1-4b60-9d60-c9dbc56b5210.png) | ![Screenshot from 2023-04-27 06-28-56](https://user-images.githubusercontent.com/83042742/234759864-74768b7d-db8b-4eda-b884-1fecd7722906.png) |

Reply:
Make the requested changes to the presentation template.